.travis.yml

```diff
@@ -31,4 +31,4 @@ script:
   - xcodebuild clean build -project "$PROJECT" -scheme "$WATCHOS_SCHEME" -destination "$WATCHOS_DESTINATION" | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-watchos.json" xcpretty -f `xcpretty-json-formatter`
   - swiftlint lint
   - bundle exec danger --verbose
-  - pod lib lint
+  - pod lib lint --skip-tests
```

Comment:
Should we do this? The podlib lint was often failing the build with timeout because it was taking too long to run. Since we already build and tests all targets before run this command, I believe we can skip build and testing for the lint and in fact only lint the podspec and subspecs. What do you guys think?
cc @omaralbeik @SD10 @guykogus

Reply:
It makes sense to me if we're performing the tests already.

Reply:
Makes sense to me